src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Ui {
+
+    public String listFoundTasks(List<Task> filteredTasks) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("Here are the matching tasks in your list:\n");
+        for (int i = 0; i < filteredTasks.size(); i++) {
+            sb.append(String.format("%d.%s\n", i + 1, filteredTasks.get(i).toString()));
+        }
```

Comment:
It probably would be good for you to have a custom toString() method for your TaskList class, so you can abstract away menial tasks like this